fill/sensors/load_cell.h

```diff
@@ -1,13 +1,20 @@
 #ifndef LOAD_CELL_H
 #define LOAD_CELL_H
 
-#define LC_CHANNEL 2
+#define LC_CHANNEL 0
```

Comment:
This doesn't make much sense to be here right? Reading from a constant like this isn't extensible